File: src/main/java/org/example/bankramenserver/infrastructure/mcp/auth/McpInitialConnectionProperties.java
package org.example.bankramenserver.infrastructure.mcp.auth;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;

@Getter
@Setter
@ConfigurationProperties(prefix = "mcp.initial-connection")
public class McpInitialConnectionProperties {

    private String connectionId;
    private String consumerId;
    private String linkedUserId;
    private boolean enabled = true;
    private String token;
}

File: src/main/kotlin/org/example/balogserver/infrastructure/mcp/auth/McpInitialConnectionProperties.kt
package org.example.balogserver.infrastructure.mcp.auth

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties(prefix = "mcp.initial-connection")
class McpInitialConnectionProperties {
    var connectionId: String? = null
    var consumerId: String? = null
    var linkedUserId: String? = null
    var enabled = true
    var token: String? = null
}
